WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageParentViewModel.kt

```diff
@@ -47,6 +51,12 @@ class PageParentViewModel
     private lateinit var site: SiteModel
     private var isStarted: Boolean = false
     private var page: PageModel? = null
+    private var _isSearchInUse = false
+
+    val filteredPages: LiveData<List<PageItem>> = mergeAsyncNotNull(defaultScope, _pages, _searchQuery) { list, query ->
```

Comment:
I think I'd still use the original name `pages` here. The View doesn't need to care if the data is filtered, it just shows whatever it gets.